javascript/webapp/src/pages/EntityReportPage.hooks.ts

```diff
@@ -95,7 +95,7 @@ export function useRelatedEntityGraphs(
 		f()
 	}, [report, relatedIds])
 
-	return useGraphDataMap(graphs)
+	return graphs !== undefined ? useGraphDataMap(graphs) : graphs
```

Comment:
you should not put a conditional in front of a hook - react always wants to execute all hooks, so it will complain. instead, you can go into the `useGraphData` hook and put a conditional inside there to return something different if `graphs` is undefined